x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -0,0 +1,512 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, useState, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiTitle,
+  EuiSpacer,
+  EuiFormRow,
+  EuiComboBox,
+  EuiKeyPadMenuItem,
+  EuiAccordion,
+  EuiButtonIcon,
+  EuiEmptyPrompt,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { HttpSetup, ToastsApi } from 'kibana/public';
+import { loadActionTypes, loadAllActions } from '../../lib/action_connector_api';
+import {
+  IErrorObject,
+  ActionTypeModel,
+  AlertAction,
+  ActionTypeIndex,
+  ActionConnector,
+  ActionType,
+} from '../../../types';
+import { SectionLoading } from '../../components/section_loading';
+import { ConnectorAddModal } from './connector_add_modal';
+import { TypeRegistry } from '../../type_registry';
+
+interface ActionAccordionFormProps {
+  actions: AlertAction[];
+  defaultActionGroupId: string;
+  setActionIdByIndex: (id: string, index: number) => void;
+  setAlertProperty: (actions: AlertAction[]) => void;
+  setActionParamsProperty: (key: string, value: any, index: number) => void;
+  http: HttpSetup;
+  actionTypeRegistry: TypeRegistry<ActionTypeModel>;
+  actionTypes?: ActionType[];
+  messageVariables?: string[];
+  defaultActionMessage?: string;
+  toastNotifications?: Pick<
```

Comment:
Similar question as the other PR -> https://github.com/elastic/kibana/pull/58514#discussion_r388318257.